pom.xml

```diff
@@ -8,10 +8,17 @@
     <artifactId>doc-samples</artifactId>
     <packaging>pom</packaging>
 
+    <!-- Parent POM defines common plugins and properties. -->
```

Comment:
This should get a license at the top and if you wish, you can get rid of the xmlns / xsi stuff.